doc/Makefile

```diff
@@ -108,6 +108,12 @@ html-noplot: sym_links_datasets glm_reports
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+html-noplot-noreport: sym_links_datasets
```

Comment:
Is this related to this PR ?

Reply:
nope: needed a faster way to build the doc locally (this does not generate the html report for maskers and glm , that usually takes some extra minutes)